src/systems/filecoin_mining/sector/sealing.id

```diff
@@ -22,20 +22,23 @@ type SealCfg struct {
 // needs to verify a Seal.
 type SealVerifyInfo struct {
     SectorID
-    OnChain     OnChainSealVerifyInfo
+    OnChain                OnChainSealVerifyInfo
     SealCfg
-    PieceInfos  [PieceInfo]
+    Randomness             SealRandomness
```

Comment:
this should be SealSeed?

Reply:
As discussed elsewhere, `SealSeed` replaces `ReplicaId`.